tooltester/README.md

```diff
@@ -1,128 +1,115 @@
-            1. => New Node ({master-floating-ip}:8080/computer/new)
-            1. Type in the name for the node and select permanent agent. Click Ok.
-            1. Fill in the \# of executors (the amount of Jenkins slaves spun up already)
-            1. Remote root directory - /home/jenkins
-            1. Host - {ip-address-slave}
-            1. If it's a new master, add credentials (Kind: SSH Username with private key, Username: jenkins, private key: <same one as before, ask around for it>)
-            1. Credentials - Jenkins
-            1. Host Key Verification Strategy - Non verifying Verification Strategy
-        1. If it's not a new master and there are already nodes, configure and set the host to the new Jenkins slave IP address then save.
-
-# Running tooltester:
-1. On your local machine, fill in ~/.tooltester/config with the appropriate values
-    1. In particular, ensure that `server-url`, `runners`, and `dockstore-version` are properly set
-A sample config file is shown below. This one will run tool tester with staging and dockstore-version 1.7.0-beta.6.
-
-Ensure that these fields are filled out correctly. Note that the dockstore-version below is for the Dockstore CLI version (not webservice)
+Now, you will want to deploy both the context to run `WDL` workflows and the context to run `CWL` workflows. To do this you
+need to be in the same directory as `agc-project.yaml`, and run the following commands (each command may take up to 20 minutes to run).
+```
+agc context deploy wdlContext
+```
+```
+agc context deploy cwlContext
+```
 
+Now, run 
 ```
-runner = cromwell cwltool cwl-runner
+agc context describe <CONTEXT NAME>
+```
+on `wdlContext` and `cwlContext`, and get the `WESENDPOINT` value from the result and append `ga4gh/wes/v1` to the end of it and place
+the resulting string in the `url` field in `WDLConfigFile` and `CWLConfigFile` respectively.
+
+Now, you will want to add your AWS profile name to both `wdlContext` and `cwlContext` in the `authorization` field. For example,
+mine is `fhembroff`.
 
-server-url = https://staging.dockstore.org/api
-jenkins-server-url = http://<find-on-collab>:8080/
-jenkins-username = <username>
-jenkins-password = <password>
-development = true
-dockstore-version = 1.7.0-beta.6
+### Finish writing Config Files
+Do the following to both `WDLConfigFile` and `CWLConfigFile`.
 
+In this step you will want to fill in the `server-url` field. This will most likely be either https://qa.dockstore.org/api or https://dockstore.org/api.
+You also need to ensure that the `server-url` field is set on your `~/.dockstore/config` file. The `server-url` must be the same across all three files.
+
+Then you will want to get your token from whatever site you chose in the above step, and fill it in, in the `token` field.
+You will need to have either admin or curator access to the appropriate dockstore site to use this command due to this [PR](https://github.com/dockstore/dockstore/pull/5357).
```

Comment:
dockstore -> Dockstore

> due to this [PR](https://github.com/dockstore/dockstore/pull/5357).

Could say that it's because the command calls endpoints that require admin or curator access.

Reply:
Fixed!